### PR TITLE
Create DropDownAlignmentBehavior.java

### PR DESCRIPTION
**Pull request by helmut8080**
Alignment for DropdownButton according to bootstrap 4

### CODE REVIEW

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import org.apache.wicket.util.lang.Args;
+
+import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.BootstrapBaseBehavior;
+import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.ICssClassNameProvider;
+import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.AlignmentBehavior.Alignment;
+import de.agilecoders.wicket.core.util.Attributes;
+
+public class DropDownAlignmentBehavior extends BootstrapBaseBehavior {
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java]: Needs javadoc

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            this.className = className;
+        }
+
+        @Override
+        public String cssClassName() {
+            return className;
+        }
+    }
+
+    private IModel<Alignment> alignment;
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java]: `final`

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java (hunk 1 above)
Comment (helmut8080) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java]: Done.

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java (hunk 2 above)
Comment (helmut8080) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java]: Done

**Pull request merged**